zer0-os/zOS
# feat: update all Button uses (props, variants, remove overriding styles, tests)

### What does this do?
- bumps zUI version v1.9.7.
- removes all overriding `Button` styles in favour of zUI set `Button` styles.
- updates `Button` props where used to align with variation updates.
- updates tests using `Button` to align with variation/prop changes.
- updates `Button` imports to direct paths (fixes some tests).

### Why are we making this change?
- due to changes to/latest version of the zUI `Button` component.
- aligns with `Button` figma design structure.

### How do I test this?
- run tests as usual.
- run the UI and manually check/test buttons.

### Key decisions and Risk Assessment:
  #### Things to consider:
  1. How will this affect security?
  1. How will this affect performance?
  1. Does this change any APIs?

Before and After screenshots
<img width="635" alt="Screenshot 2024-04-10 at 15 15 06" src="https://github.com/zer0-os/zOS/assets/39112648/d0e4b500-6a1e-4713-876e-6078c49b3ca3">
<img width="635" alt="Screenshot 2024-04-10 at 15 14 56" src="https://github.com/zer0-os/zOS/assets/39112648/6b4fd1da-e39a-496b-8f03-b4ed63e2d6a8">
<img width="635" alt="Screenshot 2024-04-10 at 14 53 19" src="https://github.com/zer0-os/zOS/assets/39112648/ae2dd12f-16d4-4c92-b901-47291f50d027">
<img width="635" alt="Screenshot 2024-04-10 at 14 53 15" src="https://github.com/zer0-os/zOS/assets/39112648/e47938dd-3565-4ee5-bbc8-a8f1dc4d2580">
<img width="283" alt="Screenshot 2024-04-10 at 14 24 48" src="https://github.com/zer0-os/zOS/assets/39112648/280196d4-41df-4b7f-bf2e-e4649af3ce03">
<img width="283" alt="Screenshot 2024-04-10 at 14 24 31" src="https://github.com/zer0-os/zOS/assets/39112648/ef4af1d7-7d86-42b2-aff6-42e8cec56182">
<img width="831" alt="Screenshot 2024-04-10 at 14 05 20" src="https://github.com/zer0-os/zOS/assets/39112648/14a0d481-a788-438e-a61e-bd081d7fb0c9">
<img width="831" alt="Screenshot 2024-04-10 at 14 05 16" src="https://github.com/zer0-os/zOS/assets/39112648/547e9159-bda5-4492-a0f0-e2ec2fabfa5f">
<img width="385" alt="Screenshot 2024-04-10 at 13 59 01" src="https://github.com/zer0-os/zOS/assets/39112648/6597d3f7-3d80-4f79-b899-c3f51a07cd1d">
<img width="385" alt="Screenshot 2024-04-10 at 13 58 46" src="https://github.com/zer0-os/zOS/assets/39112648/cd59dc64-8ec7-41f1-8d9b-27a3a0ec6d03">
<img width="689" alt="Screenshot 2024-04-10 at 13 55 05" src="https://github.com/zer0-os/zOS/assets/39112648/ae9b8078-2a15-4b96-b285-92d733d2055e">
<img width="689" alt="Screenshot 2024-04-10 at 13 54 55" src="https://github.com/zer0-os/zOS/assets/39112648/1fbb1225-bd36-4e0a-8baa-453d2a63d3cc">
<img width="373" alt="Screenshot 2024-04-10 at 13 50 59" src="https://github.com/zer0-os/zOS/assets/39112648/e23a1940-780d-4c19-984f-0001e2554991">
<img width="373" alt="Screenshot 2024-04-10 at 13 50 53" src="https://github.com/zer0-os/zOS/assets/39112648/a08e7ba4-fab9-425a-a6ec-cc341d6e67cc">
<img width="373" alt="Screenshot 2024-04-10 at 13 47 10" src="https://github.com/zer0-os/zOS/assets/39112648/36641a2b-dbf3-40db-adbe-bca08ec35a86">
<img width="373" alt="Screenshot 2024-04-10 at 13 47 05" src="https://github.com/zer0-os/zOS/assets/39112648/2930bfe1-0c1d-4aec-b035-f0fef204d909">
<img width="373" alt="Screenshot 2024-04-10 at 13 41 36" src="https://github.com/zer0-os/zOS/assets/39112648/4a131e3e-0376-44ff-a142-0d532198844b">
<img width="373" alt="Screenshot 2024-04-10 at 13 41 30" src="https://github.com/zer0-os/zOS/assets/39112648/81648a48-5513-49f1-82fd-2ecfd3f26f81">
<img width="373" alt="Screenshot 2024-04-10 at 13 38 27" src="https://github.com/zer0-os/zOS/assets/39112648/bb351fdc-8a88-4d20-8ac1-e45cabaf9112">
<img width="373" alt="Screenshot 2024-04-10 at 13 37 55" src="https://github.com/zer0-os/zOS/assets/39112648/bd49429b-1f78-4857-bb4d-da50d745ede4">
<img width="373" alt="Screenshot 2024-04-10 at 13 29 01" src="https://github.com/zer0-os/zOS/assets/39112648/eeb1923a-adf2-4c76-a1ce-c8566eb378d1">
<img width="373" alt="Screenshot 2024-04-10 at 13 28 48" src="https://github.com/zer0-os/zOS/assets/39112648/25c3e2ac-a002-4d90-8646-aafc0f7fae0e">
<img width="637" alt="Screenshot 2024-04-10 at 13 23 33" src="https://github.com/zer0-os/zOS/assets/39112648/fa70a248-4d9f-45b2-8f7b-0f7f9c5371c8">
<img width="637" alt="Screenshot 2024-04-10 at 13 23 13" src="https://github.com/zer0-os/zOS/assets/39112648/6d77e529-ca6d-422d-8dd8-84137f23b574">
<img width="345" alt="Screenshot 2024-04-10 at 13 16 05" src="https://github.com/zer0-os/zOS/assets/39112648/e8d5cc35-9843-4be6-9046-34f9682d1a87">
<img width="345" alt="Screenshot 2024-04-10 at 13 15 38" src="https://github.com/zer0-os/zOS/assets/39112648/772546b6-532e-4d43-a3d2-513ce7a79578">
<img width="345" alt="Screenshot 2024-04-10 at 13 07 50" src="https://github.com/zer0-os/zOS/assets/39112648/e48bc331-027e-4bcc-b7aa-fedd0cc059bd">
<img width="345" alt="Screenshot 2024-04-10 at 13 07 43" src="https://github.com/zer0-os/zOS/assets/39112648/f28bd838-4aed-4581-aa8e-c5876a5c21bf">
<img width="345" alt="Screenshot 2024-04-10 at 13 07 36" src="https://github.com/zer0-os/zOS/assets/39112648/bf85fa6f-c10a-41a3-b3e4-9c4fdd7a191f">
<img width="345" alt="Screenshot 2024-04-10 at 13 07 31" src="https://github.com/zer0-os/zOS/assets/39112648/ca22a92c-ead9-4da4-ac38-7dd756cd5462">
<img width="345" alt="Screenshot 2024-04-10 at 13 05 37" src="https://github.com/zer0-os/zOS/assets/39112648/37cbe96e-5389-46e7-87c0-bebd12e22378">
<img width="345" alt="Screenshot 2024-04-10 at 13 05 31" src="https://github.com/zer0-os/zOS/assets/39112648/63916a09-fe6e-4a0f-99a2-3ec976881c24">
<img width="345" alt="Screenshot 2024-04-10 at 12 59 10" src="https://github.com/zer0-os/zOS/assets/39112648/38748f79-18ac-43c1-931b-4866fc94fdef">
<img width="345" alt="Screenshot 2024-04-10 at 12 58 58" src="https://github.com/zer0-os/zOS/assets/39112648/fa5285c7-b81b-406a-823f-d30b1146ee9c">
<img width="345" alt="Screenshot 2024-04-10 at 12 53 24" src="https://github.com/zer0-os/zOS/assets/39112648/fe540f52-b158-416c-9f65-8fef792fd47f">
<img width="345" alt="Screenshot 2024-04-10 at 12 53 15" src="https://github.com/zer0-os/zOS/assets/39112648/df7e5987-3338-4f84-a6ba-fed627e3b67c">
<img width="345" alt="Screenshot 2024-04-10 at 12 49 55" src="https://github.com/zer0-os/zOS/assets/39112648/333bc70e-50d2-4ab0-837b-b5bef2ce5ee3">
<img width="345" alt="Screenshot 2024-04-10 at 12 49 31" src="https://github.com/zer0-os/zOS/assets/39112648/f6f98293-4207-4158-92c6-884bbebc43ab">
<img width="345" alt="Screenshot 2024-04-10 at 12 45 27" src="https://github.com/zer0-os/zOS/assets/39112648/ed8e806a-0e41-49cf-9104-cb5857affb05">
<img width="345" alt="Screenshot 2024-04-10 at 12 45 13" src="https://github.com/zer0-os/zOS/assets/39112648/4dc56bbf-0460-46e7-b24c-d74e4628cbf9">
<img width="345" alt="Screenshot 2024-04-10 at 12 40 51" src="https://github.com/zer0-os/zOS/assets/39112648/8e1782b6-0d87-4759-a29a-773c61d038c9">
<img width="345" alt="Screenshot 2024-04-10 at 12 40 38" src="https://github.com/zer0-os/zOS/assets/39112648/eb3a3559-01be-45d4-bf90-8b0cf55ccbb1">
<img width="345" alt="Screenshot 2024-04-10 at 12 37 00" src="https://github.com/zer0-os/zOS/assets/39112648/b32fcce5-b1a1-42d4-b7a6-6f4aaff68a11">
<img width="345" alt="Screenshot 2024-04-10 at 12 36 49" src="https://github.com/zer0-os/zOS/assets/39112648/00eedb0c-7f41-4f36-b3cd-a5b5e6d51e33">
